packages/wdio-types/src/Capabilities.ts

```diff
@@ -77,15 +77,19 @@ declare global {
             /**
              * Defines the current session’s strict file interactability.
              */
-            strictFileInteractability?: boolean,
+            strictFileInteractability?: boolean;
             /**
              * Describes the current session’s user prompt handler. Defaults to the dismiss and notify state.
              */
             unhandledPromptBehavior?: string;
             /**
              * WebDriver clients opt in to a bidirectional connection by requesting a capability with the name "webSocketUrl" and value true.
              */
-            webSocketUrl?: boolean
+            webSocketUrl?: boolean;
+            /**
+             * Maximum number of total parallel running workers.
+             */
+            maxInstances?: number;
```

Comment:
Can we create a custom interface for WebdriverIO options in capabilities, have this interface extend from it and keep it as is since we only have true w3c capabilities here.

Reply:
right on, I'll give it a shot.